docs/reference/cli-reference/cli-ref-pack.md

```diff
@@ -30,6 +30,7 @@ where `<nuspecPath>` and `<projectPath>` specify the `.nuspec` or project file,
 | --- | --- |
 | BasePath | Sets the base path of the files defined in the [.nuspec](../nuspec.md) file. |
 | Build | Specifies that the project should be built before building the package. |
+| Deterministic | Specify if the command should create a deterministic package. Multiple invocations of the pack command will generate the exact same package. The output of pack is not affected by the ambient state of the machine. Specifically, package internal psmdcp will be generated based on the package files, and the zip entries will be timestamped as 1/1/1980. |
```

Comment:
```suggestion
| Deterministic | Specify if the command should create a deterministic package. Multiple invocations of the pack command will generate the exact same package. The output of the pack command is not affected by the ambient state of the machine. In specific, package internal psmdcp will be generated based on the package files, and the zip entries will be timestamped as 1/1/1980. |
```

Reply:
I think specifically reads better than in specific. I'll make the other changes though.